deployment/secure_api_server.py

```
@@ -339,14 +341,57 @@
+    name = EMOTION_PROVIDER
+    factory = _provider_registry.get(name)
+    if not factory:
+        raise ValueError(f"Unsupported EMOTION_PROVIDER: {name}")
+    return factory()
+
+
+# Register default providers
+register_provider("hf", HFEmotionService)
+
+
+def _parse_single_text_payload(data):
```

Ruff (0.12.2)

Missing return type annotation for private function `_parse_single_text_payload`

(ANN202)